app/assets/stylesheets/finder_frontend.scss

```diff
@@ -420,11 +420,17 @@
   margin-bottom: govuk-spacing(2);
   .govuk-grid-column-two-thirds {
     text-align: right;
-    white-space: nowrap;
-
     @include govuk-media-query($from: mobile, $until: tablet) {
       text-align: left;
     }
 
   }
+
+  .gem-c-subscription-links .gem-c-subscription-links__list-item {
```

Comment:
It's not ideal to be overriding component styles. Is there any way this could be changed or added to the component itself?

Reply:
Addressed in this PR https://github.com/alphagov/govuk_publishing_components/pull/808